src/shared/components/oncoprint/controls/ConfirmNgchmModal.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
```

Comment:
would be nice to have a comment here about what NGCHM is.

Reply:
Done.  Have a look.